toolkit/tools/pkg/imagecustomizerlib/typeConversion.go

```diff
@@ -192,7 +193,7 @@ func kernelCommandLineToImager(kernelCommandLine imagecustomizerapi.KernelComman
 	}
 
 	imagerKernelCommandLine := configuration.KernelCommandLine{
-		ExtraCommandLine: string(kernelCommandLine.ExtraCommandLine),
+		ExtraCommandLine: strings.Join(kernelCommandLine.ExtraCommandLine, " "),
```

Comment:
Use the `GrubArgsToString` function to convert the list to a escaped/quoted string.